### PR TITLE
FLOE-549: modify functional mashup activity based on design crit feed…

### PR DESCRIPTION
**Pull request by dayotte**
…back
@jhung

### CODE REVIEW

File: src/documents/activities/FunctionalMashup.html.md.handlebars [hunk 1]
```diff
@@ -7,24 +7,32 @@ category: Activities
 This activity can be used in a co-design or other group design session to encourage participants to work together toward more creative and resourceful solutions, by thinking beyond the current or obvious.
 ## How
 ### Preparation
-You will need a deck of object cards. You can create your deck using any available materials. You might print images of different objects on paper or cardstock, use language learning flashcards, cut out magazine images and paste them onto cards, or find other ways to create a collection of random objects. 
+You will need a deck of object cards. You can create your deck using any available materials. You might print images of different objects on paper or cardstock, use language learning flashcards, cut out magazine images and paste them onto cards, or find other ways to create a collection of random objects.
+<figure>
+[![An image showing a large selection of cards spread out on the floor, each with a different magazine image pasted on.](/images/object_cards_thumb.jpg)](/images/object_cards_thumb.jpg)
```
Comment (jhung) [inline src/documents/activities/FunctionalMashup.html.md.handlebars]: The thumbnail is missing. Make sure to commit it.

Comment (dayotte) [reply src/documents/activities/FunctionalMashup.html.md.handlebars]: thanks! I removed the wrong file...will correct it.

File: src/documents/activities/FunctionalMashup.html.md.handlebars [hunk 2]
```diff
@@ -7,24 +7,32 @@ category: Activities
...
+* The challenge for each group is to combine and use all of the objects on their cards to create something that fulfills the selected function. One way to approach this might be to tell a story about how your objects come together to fulfill the function.
 
  **Tip**: Brainstorm different approaches and don’t limit your ideas to what is technically possible.
- **Tip**: If there are members in your group that require audio description, be sure to describe the object cards and the process as you move forward. Allow for tactile exploration where appropriate.  
- **Tip**: You can prototype or communicate your solution/concept/idea in whatever way that works best for your group (e.g. sketching, building, describing, etc).   
- **Tip**: In addition to the object cards, you can use the available materials to prototype your idea.     
+
+ **Tip**: If there are members in your group that require audio description, be sure to describe the object cards and the process as you move forward. Allow for tactile exploration where appropriate.
```
Comment (jhung) [inline src/documents/activities/FunctionalMashup.html.md.handlebars]: There are 4 tips in a row. Would it make sense to just have a single **Tip**: and have a list below it?

Comment (dayotte) [reply src/documents/activities/FunctionalMashup.html.md.handlebars]: Maybe... but I do like the individual emphasis on each one. We have this in other places too (like Collaborative Art Activity). In some cases the tips are scattered throughout the text and in other cases they are all together like this. I think these particular ones started out scattered throughout the bullet points but it seemed clearer/cleaner to me to put them all at the end of the list.

**Pull request merged**